PWGCF/FEMTOSCOPY/macros/AddTaskFemtoNanoGrandma.C

```diff
@@ -7,13 +7,16 @@
 #include "AliFemtoDreamCascadeCuts.h"
 #include "AliFemtoDreamCollConfig.h"
 
-AliAnalysisTaskSE *AddTaskFemtoNanoGrandma(bool fullBlastQA = false, int phiSpinning =
-                                         0,
-                                     int nSpins = 1, double corrRange = 0.1,
+AliAnalysisTaskSE *AddTaskFemtoNanoGrandma(bool fullBlastQA = false,
+									 int phiSpinning =0,
+                                     int nSpins = 1,
+									 double corrRange = 0.1,
                                      bool Systematic = false,
-                                     const char *cutVariation = "0") {
+                                     const char *cutVariation = "0",
+									 const char *sTcut = "8") {
```

Comment:
I am not so sure about this - I would rather make this the second-to-last argument (since there are only 8 variations) and use the wagon configuration to take care of all the systematic variations.
Also, below you compare "8" == "08" which I don't think is going to work